Misc/NEWS.d/next/Library/2020-10-02-17-35-19.bpo-33533.GLIhM5.rst

```diff
@@ -0,0 +1,2 @@
+:func:`asyncio.as_completed` now returns an asynchronous iterator.  Patch by
+Justin Arthur.
```

Comment:
Maybe clarify that the iterator is ambidextrous (or polymorphic or whatever :-).